microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-operators-olm"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* Use link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/cli_tools/opm-cli#cli-opm-install[`opm` CLI] for use with network-connected clusters.
+* Install the link:https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-disconnected.html#installation-oc-mirror-installing-plugin_installing-mirroring-disconnected[the oc-mirror OpenShift CLI plugin] for use with disconnected clusters.
```

Comment:
@pmtk My understanding is that users must create catalogs to use with MicroShift, so they can create them by using either path, BUT the `opm` CLI does not work with disconnected clusters. If that's the case, then I would think using `oc mirror` for disconnected is mandatory? I am foggy on the nuance of this as well. The `oc mirror` docs are huge, so we need to whittle down to the links our users will need.

Reply:
I think it kind of depends where you push your operators, bundles, catalogs, and anything related to OLM.
If user has its own operator and catalog then, even for disconnected networks, they could build it on site and push directly to their registry, they would need `opm` to create bundles, catalogs, etc. even though they're disconnected.

At least that's my understanding of it but I might be nitpicking or focusing too much on the trees rather than woods.